flake.nix

```diff
@@ -112,40 +150,39 @@
         cargo-clippy = craneLib.cargoClippy (commonArgs // {
           inherit cargoArtifacts;
 
-          cargoClippyExtraArgs = "--all-targets --workspace -- -D warnings";
-
-          doCheck = true;
+          cargoClippyExtraArgs = "--all-targets -- -D warnings";
```

Comment:
```suggestion
          cargoClippyExtraArgs = "--all-targets --workspace -- -D warnings";
```

We should still run clippy over the whole workspace for now.

Reply:
I tested and I'm pretty sure this is how `--all-targets` works.

Reply:
![image](https://user-images.githubusercontent.com/15848336/233741078-add1390f-9891-441e-aa21-fb187ab28893.png)
This doesn't seem to relate to workspaces though

Reply:
> By default, when no package selection options are given, the packages selected depend on the selected manifest file (based on the current working directory if --manifest-path is not given). If the
       manifest is the root of a workspace then the workspaces default members are selected, otherwise only the package defined by the manifest will be selected.

Seems that it selects all our members and that this was only something to think about when we had excluded members. Yeah?

Reply:
Yeah, I guess it wouldn't pick a random package if we don't have a default so it must be doing all of them atm.